src/dali/plugin/pair_converter/csv/kraken.py

```diff
@@ -222,7 +231,10 @@ def _retrieve_cached_bar(self, base_asset: str, quote_asset: str, timestamp: int
                             close=RP2Decimal(row[self.__CLOSE]),
                             volume=RP2Decimal(row[self.__VOLUME]),
                         )
-
+            except FileNotFoundError:
```

Comment:
Were you having trouble with missing files? If chunked files are missing that means there is something wrong with the chunking process and the recording of it that needs to be fixed. This is just going to hide that bug.

Reply:
I believe at this point I have explained multiple times 'BTC_OHLCVT.zip' doesn't exist.

Reply:
I know that it isn't on the Google Drive. I guess it wasn't clear it wasn't in the chunked files, but I guess it makes sense.

Reply:
OK, let's leave it for now and I'll look at it when I implement aliases.